quinn-udp/tests/tests.rs

```diff
@@ -31,6 +31,29 @@ fn basic() {
     );
 }
 
+#[test]
+fn basic_src_ip_ipv6() {
+    let send = UdpSocket::bind((Ipv6Addr::LOCALHOST, 0))
+        .or_else(|_| UdpSocket::bind((Ipv4Addr::LOCALHOST, 0)))
```

Comment:
If this test is IPv6 specific, do these `or_else` clauses make sense?

Reply:
Ah that was a left over, it is not IPv6 speciifc. As soon as we add more than the initial cmsg, this will break.